modules/install-sno-installing-with-the-assisted-installer.adoc

```diff
@@ -0,0 +1,25 @@
+// This is included in the following assemblies:
+//
+// installing_sno/install-sno-installing-sno.adoc
+
+:_content-type: PROCEDURE
+[id="install-sno-installing-with-the-assisted-installer_{context}"]
+= Installing {sno} with the Assisted Installer
+
+Use the Assisted Installer to install the single-node cluster.
+
+.Procedure
+
+. Attach the {op-system} discovery ISO to the target host.
+
+. Configure the boot drive order in the server BIOS settings to boot from the attached discovery ISO and then reboot the server.
+
+. On the administration host, return to the browser and refresh the page. Wait for the host to appear in the list of discovered hosts. If necessary, reload the link:https://console.redhat.com/openshift/assisted-installer/clusters[*Assisted Clusters*] page and select the cluster name.
+
+. Complete the install wizard steps. Add networking details, including a subnet from the available subnets. Add the SSH public key if necessary.
+
+. Monitor the installation's progress. Watch the cluster events. After the installation process finishes writing the discovery image to the server's drive, the server restarts.
+
+. Remove the discovery ISO, and reset the BIOS to boot to the server's newly installed hard disk.
++
+The server restarts several times, deploying the control plane.
```

Comment:
```suggestion
The server may restart several times, deploying the control plane.
```

To emphasize it's an automatic process

Reply:
This wording doesn't imply an automatic process. it implies that there is a possibility that the server will restart.

Reply:
"The server restarts several times automatically, deploying the control plane."